ament_uncrustify/ament_uncrustify/configuration/ament_code_style.cfg

```diff
@@ -1,1752 +1,2110 @@
 
 # Indent OC blocks in a message relative to the parameter name.
 # 0=use indent_oc_block rules, 1+=spaces to indent
-indent_oc_block_msg                       = 0        # number
+indent_oc_block_msg             = 0        # unsigned number
 
 # Minimum indent for subsequent parameters
-indent_oc_msg_colon                       = 0        # number
+indent_oc_msg_colon             = 0        # unsigned number
 
-# If true, prioritize aligning with initial colon (and stripping spaces from lines, if necessary).
-# Default is true.
-indent_oc_msg_prioritize_first_colon      = true     # false/true
+# If True, prioritize aligning with initial colon (and stripping spaces from lines, if necessary).
+# Default=True.
+indent_oc_msg_prioritize_first_colon = true     # false/true
 
 # If indent_oc_block_msg and this option are on, blocks will be indented the way that Xcode does by default (from keyword if the parameter is on its own line; otherwise, from the previous indentation level).
-indent_oc_block_msg_xcode_style           = false    # false/true
+indent_oc_block_msg_xcode_style = false    # false/true
 
 # If indent_oc_block_msg and this option are on, blocks will be indented from where the brace is relative to a msg keyword.
-indent_oc_block_msg_from_keyword          = false    # false/true
+indent_oc_block_msg_from_keyword = false    # false/true
 
 # If indent_oc_block_msg and this option are on, blocks will be indented from where the brace is relative to a msg colon.
-indent_oc_block_msg_from_colon            = false    # false/true
+indent_oc_block_msg_from_colon  = false    # false/true
 
 # If indent_oc_block_msg and this option are on, blocks will be indented from where the block caret is.
-indent_oc_block_msg_from_caret            = false    # false/true
+indent_oc_block_msg_from_caret  = false    # false/true
 
 # If indent_oc_block_msg and this option are on, blocks will be indented from where the brace is.
-indent_oc_block_msg_from_brace            = false    # false/true
+indent_oc_block_msg_from_brace  = false    # false/true
 
 # When identing after virtual brace open and newline add further spaces to reach this min. indent.
-indent_min_vbrace_open                    = 0        # number
+indent_min_vbrace_open          = 0        # unsigned number
+
+# True: When identing after virtual brace open and newline add further spaces after regular indent to reach next tabstop.
+indent_vbrace_open_on_tabstop   = false    # false/true
+
+# If True, a brace followed by another token (not a newline) will indent all contained lines to match the token.Default=True.
+indent_token_after_brace        = false    # false/true
 
-# TRUE: When identing after virtual brace open and newline add further spaces after regular indent to reach next tabstop.
-indent_vbrace_open_on_tabstop             = false    # false/true
+# If True, cpp lambda body will be indentedDefault=False.
+indent_cpp_lambda_body          = true     # false/true
```

Comment:
This will indent the body of lambdas by one level which makes them fit in much nicer with the surrounding code.